common-react/@dbeaver/ui-kit/src/Checkbox/Checkbox.css

```diff
@@ -0,0 +1,98 @@
+@import './_base.css';
+@import './_sizes.css';
+
+@layer base {
+  .dbv-kit-checkbox {
+    display: inline-flex;
+    user-select: none;
+    align-items: start;
+    min-height: var(--dbv-kit-checkbox-height);
+    gap: var(--dbv-kit-checkbox-gap);
+    line-height: var(--dbv-kit-checkbox-height);
+    font-size: var(--dbv-kit-checkbox-font-size);
+    color: var(--dbv-kit-checkbox-foreground);
+    outline: var(--dbv-kit-color-outline);
+
+    &:hover {
+      cursor: pointer;
+
+      .dbv-kit-checkbox__check {
+        border-color: var(--dbv-kit-checkbox-border-color-checked);
+        box-shadow: 0 0 0 2px var(--dbv-kit-checkbox-hover-shadow-color);
+      }
+    }
+
+    &[data-checked='true'] {
+      border-color: var(--dbv-kit-checkbox-border-color-checked);
+    }
+
+    &[data-disabled='true'] {
+      pointer-events: none;
+      opacity: var(--dbv-kit-control-disabled-opacity);
+    }
+  }
+
+  .dbv-kit-checkbox__check {
+    margin-top: calc(var(--dbv-kit-checkbox-height) * (calc(1 - var(--dbv-kit-checkbox-check-size)) / 2));
+    height: calc(var(--dbv-kit-checkbox-height) * var(--dbv-kit-checkbox-check-size));
+    line-height: calc(var(--dbv-kit-checkbox-height) * var(--dbv-kit-checkbox-check-size));
+    width: calc(var(--dbv-kit-checkbox-height) * var(--dbv-kit-checkbox-check-size));
+    min-width: calc(var(--dbv-kit-checkbox-height) * var(--dbv-kit-checkbox-check-size));
+    border-radius: var(--dbv-kit-checkbox-border-radius);
+    background-color: var(--dbv-kit-checkbox-background);
+    border-color: var(--dbv-kit-checkbox-border-color);
+    border-width: var(--dbv-kit-checkbox-border-width);
+    border-style: var(--dbv-kit-checkbox-border-style);
+    color: var(--dbv-kit-checkbox-icon-color);
+    outline-offset: var(--dbv-kit-outline-offset);
+    outline-width: var(--dbv-kit-control-outline-width);
+    outline-color: var(--dbv-kit-color-outline);
+    transition-property: stroke-dashoffset;
+    transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+    transition-duration: 150ms;
+    stroke-dasharray: 20;
+    stroke-dashoffset: 20;
+
+    &[data-focus-visible] {
```

Comment:
Should we add `&:focus-visible,` as you did for input and button?

Reply:
No, it's artificial node and can't get any state except `:hover`. Only parent can be :focus-visible